Sources/Errors/UIAlertController+Error.swift

```diff
@@ -10,19 +10,32 @@ import UIKit
 
 public extension UIAlertController {
     convenience init(error: Error, preferredStyle: UIAlertController.Style = .alert) {
-        switch error {
-        case let error as ResolvableError:
-            self.init(title: error.errorDescription ?? NSLocalizedString("Error", comment: "Error"), message: error.failureReason ?? error.localizedDescription, preferredStyle: preferredStyle)
+        let (title, message) = UIAlertController.alertTitleMessage(error: error)
+        self.init(title: title, message: message, preferredStyle: preferredStyle)
+        if let error = error as? ResolvableError {
             error.actions.map { $0.alertAction() }.forEach(self.addAction)
-            if error.actions.isEmpty {
-                self.addAction(UIAlertAction(title: NSLocalizedString("OK", comment: "OK"), style: .default))
+            if !error.actions.isEmpty {
+                return
             }
+        }
+        self.addAction(UIAlertAction(title: UIAlertController.okButtonText, style: .default))
+    }
+
+    private static func alertTitleMessage(error: Error) -> (String, String) {
+        switch error {
         case let error as LocalizedError:
-            self.init(title: error.errorDescription ?? NSLocalizedString("Error", comment: "Error"), message: error.failureReason ?? error.localizedDescription, preferredStyle: preferredStyle)
-            self.addAction(UIAlertAction(title: NSLocalizedString("OK", comment: "OK"), style: .default))
+            return (error.errorDescription ?? defaultErrorText,
+                    error.failureReason ?? error.localizedDescription)
         default:
-            self.init(title: NSLocalizedString("Error", comment: "Error"), message: error.localizedDescription, preferredStyle: preferredStyle)
-            self.addAction(UIAlertAction(title: NSLocalizedString("OK", comment: "OK"), style: .default))
+            return (defaultErrorText, error.localizedDescription)
         }
     }
+
+    private static var defaultErrorText: String {
```

Comment:
It only adds keywords and at the [calling place](https://github.com/thefuntasty/FuntastyKit/pull/50/files/d0f32e36611ef207e030ce290ea8a4ee55c54077#diff-b3da0a95de253fe04724087d49e230e4R21) it is longer. Basically, i see no reason for these properties to be `static`.

Reply:
@mkj-is if I will declare these properties as non-static I will be not able to compile code of this class, because I use these static properties in the `init` function (or in functions that are calling from the `init`).